src/xdocs/cmdline.xml.vm

```diff
@@ -169,27 +169,32 @@ java -D&lt;property&gt;=&lt;value&gt;  \
     <section name="Download and Run">
       <p>
         It is possible to run Checkstyle directly from the JAR file using
-        the <code>-jar</code> option. Download latest
+        the <code>-jar</code> option.
+      </p>
+      <p>
+        Follow the steps mentioned below to run Checkstyle locally:
+      </p>
+      <p>
+        1. Download latest
         <a href="https://github.com/checkstyle/checkstyle/releases/download/checkstyle-${projectVersion}/checkstyle-${projectVersion}-all.jar">
-        checkstyle-${projectVersion}-all.jar</a>.
-        An example of run would be:
+           checkstyle-${projectVersion}-all.jar</a>.
+      </p>
+      <p>
+        2. It is recommended to use configuration files that are embedded in jar files, but latest
+             configuration files are there:
+        <a href="https://raw.githubusercontent.com/checkstyle/checkstyle/master/src/main/resources/sun_checks.xml">
+          sun_checks.xml</a>
+        <a href="https://raw.githubusercontent.com/checkstyle/checkstyle/master/src/main/resources/google_checks.xml">google_checks.xml</a>
+      </p>
+      <p>
+        3. Run Checkstyle using either of the commands:
       </p>
       <div class="wrap-content">
         <source>
           java -jar checkstyle-${projectVersion}-all.jar -c /sun_checks.xml MyClass.java
           java -jar checkstyle-${projectVersion}-all.jar -c /google_checks.xml MyClass.java
         </source>
       </div>
-      <p>
```

Comment:
Keep recommendation after first execution, no need to overwhelm user with details at the beginning